examples/defaulter-gen/output_tests/wholepkg/vendor/k8s.io/apimachinery/pkg/runtime/mock.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
In that case, I suggest you rename the expected file zz_generated.go to zz_generated.expected, add the name zz_generated.go to git_ignore of that folder, and do a `diff zz_generated.expected zz_generated.go` instead of `git diff ...`.